### ISSUE TITLE
Partial synchronization on modification

### ISSUE THREAD

**Issue by wdomski** [nextcloud/desktop#46]
This is a possible feature request for the desktop (Windows) Nextcloud sync application.

Let's say that I have my cloud directory in a local path C:/Nextcloud. When I create/add/remove/modify a file/directory in a path i.e. C:/Nextclud/Some/Nested/Path the Nextcloud sync app is scanning whole root directory (C:/Nextcloud) for changes, it can take some time and consume a lot of CPU resources for big clouds. Can you detect where the modification had occurred and rescan only this part of local cloud directory?

It would be nice to have Nextcloud scan whole root directory only once when the app is starting up but not every time something is modified inside.

**Comment by guruz** [comment]
This is done upstream in 2.5
https://github.com/owncloud/client/issues/2297

**Comment by wdomski** [comment]
In what version of Nextcloucd client will it be introduced? Could you also refer me to some schedule for launching new versions of Nextcloud client?

**Comment by Norrodar** [comment]
Are there any new updates? ^^

**Comment by camilasan** [comment]
Could you try the latest desktop client? There has been significant changes in the sync engine since this issue was opened.
If you think the issue persists with the latest client, please re-open the issue.

**Comment by camilasan** [comment]
The latest version can be seen by checking [https://nextcloud.com/install/#install-clients](https://nextcloud.com/install/#install-clients) and [https://nextcloud.com/install/#instructions-server](https://nextcloud.com/install/#instructions-server) respectively.

Thanks!
